internal/topology/variables/clusterclass_variable_validation.go

```diff
@@ -136,7 +137,7 @@ func validateRootSchema(clusterClassVariable *clusterv1.ClusterClassVariable, fl
 	}
 
 	// Validate defaults in the structural schema.
-	validationErrors, err := structuraldefaulting.ValidateDefaults(fldPath.Child("schema"), ss, true, true)
+	validationErrors, err := structuraldefaulting.ValidateDefaults(context.TODO(), fldPath.Child("schema"), ss, true, true)
```

Comment:
Can you please check if we might be able to pass through the context? (given that it's at least an internal func here it might be possible

Reply:
I tried one implementation by passing in context through to the `ValidateClusterClassVariables`. I am not very happy passing in context through so many functions but most of them are internal.

Reply:
Looks like they are all internal (if I didn't miss anything)

=> lgtm! Thx